src/main/resources/admin/tools/content-type-manager/components/form-definition/component-form-definition-allow-child-content-types.html

```diff
@@ -14,6 +14,7 @@
                   v-model="allowedChildContentTypes[index].elements[0].text"
                   :label="fields.label"
                   :hint="fields.hint"
+                  :rules="[rules.requiredText, rules.noSpaces]"
```

Comment:
Required? I don't think allow-child-content-types is required. It can be added to the `<config>`.